contracts/src/base/registry/facets/checker/EntitlementChecker.sol

```diff
@@ -231,7 +228,7 @@ contract EntitlementChecker is IEntitlementChecker, Facet {
     return
       uint256(
         keccak256(
-          abi.encodePacked(block.prevrandao, block.timestamp, seed, msg.sender)
+          abi.encode(block.prevrandao, block.timestamp, seed, msg.sender)
```

Comment:
why encode? we don't care abount maintaining type information on these values.

Reply:
Because `abi.encodePacked` involves bit shifting and is more expensive.